script/openqa-bootstrap

```diff
@@ -16,15 +19,15 @@ zypper -n install --no-recommends openQA-local-db apache2 openQA-worker qemu-kvm
 
 # setup database
 systemctl enable --now postgresql
-su postgres -c "createuser -D geekotest"
-su postgres -c "createdb -O geekotest openqa"
+getent passwd $dbuser || su postgres -c "createuser -D $dbuser"
+su postgres -c "psql -lqt" | cut -d \| -f 1 | grep -qw $dbname || su postgres -c "createdb -O $dbuser $dbname"
```

Comment:
I think this line is too complicated - it should be put into a separate `if` statement and get a comment what is done here.